aptos-move/framework/src/release_bundle.rs

```diff
@@ -205,4 +208,79 @@ impl ReleasePackage {
         }
         order.push(id)
     }
+
+    pub fn generate_script_proposal(&self, out: PathBuf) -> anyhow::Result<()> {
+        let writer = CodeWriter::new(Loc::default());
+        emitln!(
+            writer,
+            "// Upgrade proposal for package `{}`\n",
+            self.metadata.name
+        );
+        emitln!(
+            writer,
+            "/*\nBuildInfo.yaml:\n{}\n*/\n",
+            self.metadata.build_info
+        );
+        emitln!(writer, "script {");
+        writer.indent();
+        emitln!(writer, "use aptos_framework::aptos_governance;");
+        emitln!(writer, "use aptos_framework::code;\n");
+        emitln!(writer, "fun main(proposal_id: u64){");
+        writer.indent();
+        emitln!(
+            writer,
+            "let framework_signer = aptos_governance::resolve(proposal_id, @aptos_framework);"
```

Comment:
I think we should leave a TODO here to allow for retrieving a different signer (e.g. 0x2 for upgrading Token)

Reply:
I added the address as a parameter to the CLI command.

Reply:
Awesome!